README.Rmd

```diff
@@ -73,6 +88,55 @@ shinyApp(ui, server)
 knitr::include_graphics("man/figures/hierplane_spacyr.png")
 ```
 
+If you are familiar with `data.tree`, you can use it to visualize trees as well. Here is an example of generating a hierplane from YAML:
+
+```r
+library(data.tree)
+library(yaml)
+
+"
+name: r4fun
+tyler:
+  name: Tyler
+  job: Data Scientist
+  species: Human
+  toulouse:
+    name: Toulouse
+    job: Systems Engineer
+    species: Cat
+    jojo:
+      name: Jojo
+      job: Python Programmer
+      species: Dog
+  ollie:
+    name: Ollie
+    job: Database Administrator
+    species: Dog
+  lucas:
+    name: Lucas
+    job: R Programmer
+    species: Rabbit
+" -> yaml
+
+yaml %>% 
+  yaml.load() %>% 
+  as.Node() %>% 
+  hp_datatree(
+    title = "r4fun github group",
+    link = "species",
+    attributes = "job"
+  ) %>% 
+  hierplane(
+    theme = "light", 
+    width = "auto",
+    height = "auto"
+  )
+```
+
+```{r, echo=FALSE, dpi=300}
+knitr::include_graphics("man/figures/hierplane_datatree.png")
+```
+
 While hierarchical data isn't common in a `data.frame` centric language like R, we are working on a way to parse a `data.frame` to hierplane ready data. This works by using `hp_dataframe()`:
```

Comment:
suggest to change from `we are working on a way` to `we have developed a way` since it's already done :D

Reply:
Thanks, I'll make this change.